servers/rendering/renderer_rd/forward_clustered/scene_shader_forward_clustered.cpp

```diff
@@ -766,6 +766,12 @@ void SceneShaderForwardClustered::init(const String p_defines) {
 		actions.render_mode_defines["debug_shadow_splits"] = "#define DEBUG_DRAW_PSSM_SPLITS\n";
 		actions.render_mode_defines["fog_disabled"] = "#define FOG_DISABLED\n";
 
+		bool specular_occlusion = GLOBAL_GET("rendering/reflections/specular_occlusion/enabled");
+		if (specular_occlusion) {
+			// If specular_occlusion == false, this will be defined already.
```

Comment:
Same comment as above, but note that if you want to keep the check in, then the compatibility code needs to change to match this (i.e. checking `if (specular_occlusion)` instead of `if (!specular_occlusion)`